cypress/utils/dashboards/remove_workspace.js

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export function removeSampleDataAndWorkspace(url, workspaceName) {
```

Comment:
this seems like util functions but they are tests. if this helper functions fail does the entire test fail?

if so this seems wrong. we shouldn't use E2E testing if we are doing test setup. it's too slow and flaky. we should use the saved objects api which workspaces supports and just execute queries.

Reply:
we don't use this one. it is for easy migration.